test/integration_test/run.sh

```diff
@@ -41,12 +41,18 @@ fi
 
 if [ "$test_case" == "*" ]; then
     for script in $test_dir/$test_case/run.sh; do
+        # jvmchaos and chaosd are not supported in aarch64
+        # TODO: support JVMChaos / chaosd in aarch64, and remove this check
+        if [[ ($script == *"jvm"* || $script == *"physical_machine"*) && "$(uname -m)" == "aarch64" ]]; then
+            continue
+        fi
+
         run $script
     done
 else
     for name in $test_case; do
         script="$test_dir/$name/run.sh"
-	echo "run $script"
+	    echo "run $script"
```

Comment:
can remove the space

Reply:
I'll also replace the initial tab with spaces (as this file is indented with spaces), to keep align with former and later codes.